modules/build/src/main/scala/scala/build/interactive/Interactive.scala

```diff
@@ -0,0 +1,48 @@
+package scala.build.interactive
+
+import scala.build.Logger
+import scala.build.errors.BuildException
+import scala.build.options.BuildOptions
+import scala.io.StdIn.readLine
+
+case class Interactive(options: BuildOptions, logger: Logger) {
+
+  private val isInteractive = options.internal.interactive.getOrElse(false)
+
+  def chooseOne[E <: BuildException](
+    msg: String,
+    options: List[String],
+    fallbackError: E
+  ): Either[E, String] =
+    if (isInteractive && coursier.paths.Util.useAnsiOutput()) {
+      logger.message(msg)
+      options.zipWithIndex.foreach {
+        case (option, index) => logger.message(s"[$index] $option")
+      }
```

Comment:
We should probably print directly to `System.err` here… Because we read directly from stdin anyway (via `readLine`), and not printing anything before waiting for input would be a problem.